src/utils/image.ts

```diff
@@ -121,3 +130,10 @@ export const determineAvatar: DetermineAvatar = (options) => {
 
   return list[hash % list.length];
 };
+
+const makeProfileImageLoadErrorHandler = (
+  src: string
+): React.ReactEventHandler<HTMLImageElement> => ({ currentTarget }) => {
+  currentTarget.onerror = null; // prevents looping
```

Comment:
Should an error be at least logged to console (in case the browser already doing it)?
Is it an important one to also involve Bugsnag? (if yes, we can just put a `@debt use Bugsnag` comment for now)

Reply:
not sure if it's important... it'll be errored if our default image is missing. 
Anyway, added a debt comment.